.prettierrc

```diff
@@ -1,5 +1,4 @@
 {
-    "singleQuote": true,
-    "parser": "typescript",
```

Comment:
Prettier will automatically detect the proper `parser` by the file extension, so we don't need to explicitly set this.